_helpers_/common.ts

```diff
@@ -0,0 +1,8 @@
+export function generateRandomString() {
+    return Math.random().toString(36).substring(3, 10);
+}
+
+export function generateRandomType() {
```

Comment:
naming: What type? Pixel Type or something else should be better.

Reply:
fixed